web/src/components/search/SearchResult.vue

```diff
@@ -478,6 +483,10 @@ export default defineComponent({
 
             // rerender the chart
             nextTick(() => {
+              if(!res.data.aggregations){
+                console.log("res.data.aggregations is null")
```

Comment:
You should have `;` at the end to keep format.
And you should have a blank space before `{` to keep format.